kernel: bump 5.4 to 5.4.84

graysky2 opened the pull request:
All modifications made by `update_kernel.sh`

Build system: x86_64
Build-tested: ipq806x/R7800, ath79/generic, bcm27xx/bcm2711
Run-tested: ipq806x/R7800

No dmesg regressions, everything functional

Signed-off-by: John Audia <graysky@archlinux.us>

graysky2 committed:
kernel: bump 5.4 to 5.4.84

All modifications made by update_kernel.sh

Build system: x86_64
Build-tested: ipq806x/R7800, ath79/generic, bcm27xx/bcm2711
Run-tested: ipq806x/R7800

No dmesg regressions, everything functional

Signed-off-by: John Audia <graysky@archlinux.us>

adschm commented:
I get this when refreshing ath79:

```
diff --git a/target/linux/ath79/patches-5.4/450-fix-block-protection-clearing.patch b/target/linux/ath79/patches-5.4/450-fix-block-protection-clearing.patch
index a7ed62302d..dc76161892 100644
--- a/target/linux/ath79/patches-5.4/450-fix-block-protection-clearing.patch
+++ b/target/linux/ath79/patches-5.4/450-fix-block-protection-clearing.patch
@@ -17,12 +17,12 @@ Signed-off-by: Nick Hainke <vincent@systemli.org>

 --- a/drivers/mtd/spi-nor/spi-nor.c
 +++ b/drivers/mtd/spi-nor/spi-nor.c
-@@ -1981,7 +1981,7 @@ static int sr2_bit7_quad_enable(struct s
+@@ -1985,7 +1985,7 @@ static int sr2_bit7_quad_enable(struct s
  static int spi_nor_clear_sr_bp(struct spi_nor *nor)
  {
        int ret;
 -      u8 mask = SR_BP2 | SR_BP1 | SR_BP0;
 +      u8 mask = SR_TB | SR_BP2 | SR_BP1 | SR_BP0;

-       ret = read_sr(nor);
-       if (ret < 0) {
+       if (nor->flags & SNOR_F_HAS_4BIT_BP)
+               mask |= SR_BP3;
```

(Already got this based on your 5.4.83 patch, but ignored it then since it doesn't break anything.)

Who is wrong?

graysky2 commented:
:confused: My fork of master is currently at `bfc433e` ... is that where I ran `./update_kernel.sh -v -u 5.4 5.4.84` to generate this commit.  I will repeat but I think it will arrive at the identical code.

graysky2 commented:
@adschm - The 2nd update finished, no modification to ath79.

adschm commented:
Interesting, that's what I feared. I will investigate.

curtdept reviewed (approved):
Pending the identified ath79 patch weirdness.

Build and run/running tested x86_64: Curtis Deptuck <curtdept@me.com>

graysky2 commented:
@adschm - Just as a triple check, I did a fresh clone of the github repo and ran the update.  I got a match to what you got:
```
diff --git a/target/linux/ath79/patches-5.4/450-fix-block-protection-clearing.patch b/target/linux/ath79/patches-5.4/450-fix-block-protection-clearing.patch
index a7ed62302d..dc76161892 100644
--- a/target/linux/ath79/patches-5.4/450-fix-block-protection-clearing.patch
+++ b/target/linux/ath79/patches-5.4/450-fix-block-protection-clearing.patch
@@ -17,12 +17,12 @@ Signed-off-by: Nick Hainke <vincent@systemli.org>

 --- a/drivers/mtd/spi-nor/spi-nor.c
 +++ b/drivers/mtd/spi-nor/spi-nor.c
-@@ -1981,7 +1981,7 @@ static int sr2_bit7_quad_enable(struct s
+@@ -1985,7 +1985,7 @@ static int sr2_bit7_quad_enable(struct s
  static int spi_nor_clear_sr_bp(struct spi_nor *nor)
  {
    int ret;
 -  u8 mask = SR_BP2 | SR_BP1 | SR_BP0;
 +  u8 mask = SR_TB | SR_BP2 | SR_BP1 | SR_BP0;

-   ret = read_sr(nor);
-   if (ret < 0) {
+   if (nor->flags & SNOR_F_HAS_4BIT_BP)
+     mask |= SR_BP3;
```
I cannot explain how the two (my "legacy" fork of openwrt and this new one) are different.

adschm commented:
Do you delete tmp (rm -rf tmp) before refreshing?

graysky2 commented:
I did delete `tmp/*` before refreshing.
I always update before running the script.
I do not have local patches.

I am going to delete my fork, refork, and try it again.  So bizarre.

adschm commented:
Running out of ideas. Note that I am using a repo that only sees a "make distclean" about once a year, and clean up stuff only when really necessary, So, it's interesting that my refresh went different.

graysky2 commented:
@adschm - This is getting ridiculous.  I deleted my github fork, re-forked, cloned to my local machine (`git clone git@github.com:graysky2/openwrt.git`) then finally ran the update script, the result matches the result in this PR.  By contrast, when I just directly cloned (`git clone https://github.com/openwrt/openwrt.git`), and ran the update script, the result is different.

[Result](https://gist.github.com/graysky2/32623294d3481344ac2bdfcb543b1e8e) from cloning my fork.
[Result](https://gist.github.com/graysky2/4900d519193dbac2f7c4cc17b580b394) from directly cloning openwrt (via https not git@github).

graysky2 commented:
I get a different result when I clone the repo using git vs https.  `git clone git@github.com:openwrt/openwrt.git` then the update script resulted in [this](https://gist.github.com/graysky2/4f0b895f0fae2a150382982497cab769) patch.

graysky2 commented:
I get a different result when I clone the repo using git vs https.  `git clone git@github.com:openwrt/openwrt.git` then the update script resulted in [this](https://gist.github.com/graysky2/4f0b895f0fae2a150382982497cab769) patch.

As you see, the update script also did not update `include/kernel-version.mk` with the 0.84 info.

shell history:
```
10175  git clone git@github.com:openwrt/openwrt.git
10176  cd openwrt
10177  cp /incoming/openwrt/update_kernel.sh .
10178  ./update_kernel.sh -v -u 5.4 5.4.84
10179  gitc
```

graysky2 commented:
I get a different result when I clone the repo using git vs https.  `git clone git@github.com:openwrt/openwrt.git` then the update script resulted in [this](https://gist.github.com/graysky2/4f0b895f0fae2a150382982497cab769) patch.

As you see, the update script also did not update `include/kernel-version.mk` with the 0.84 info.  I am not sure but when I prepared this PR, I might have omitted the `5.4.84` token on the call to `update-kernel.sh`

shell history for this recent experiment:
```
10175  git clone git@github.com:openwrt/openwrt.git
10176  cd openwrt
10177  cp /incoming/openwrt/update_kernel.sh .
10178  ./update_kernel.sh -v -u 5.4 5.4.84
10179  gitc
```

I am repeating the clone via https and line 10178 from my history above now.

graysky2 commented:
Fork deleted.  Forked again, cloned, the update script requires `staging_dir/host/bin/sed` so copy my default config over, `make prepare` then `./update_kernel.sh -v -u 5.4 5.4.84` ... after it finished [here](https://gist.github.com/graysky2/52ebcd08605f5f5bfb711e97d3a4a8c7) is the result.

It is the same as my initial PR.  Both of these differ from when I just cloned the repo and ran the update.

graysky2 commented:
Experiment 1) Fork deleted.  Forked again, cloned, ran `./scripts/feeds update -a && ./scripts/feeds install -a` the update script requires `staging_dir/host/bin/sed` so copy my default config over, `make prepare` then `./update_kernel.sh -v -u 5.4 5.4.84` ... after it finished [here](https://gist.github.com/graysky2/52ebcd08605f5f5bfb711e97d3a4a8c7) is the result.  **It is the same as my initial PR.** 

Experiment 2) Just to show that I am not crazy, I repeated the more simplistic clone and update I posted a few above.  `git clone git@github.com:openwrt/openwrt.git && cd openwrt` then copy the update script over and run it `./update_kernel.sh -v -u 5.4 5.4.84` (nothing else) and it is different.  I do not understand.  Has to be something the extra setup is doing, no?
```
diff --git a/target/linux/ath79/patches-5.4/450-fix-block-protection-clearing.patch b/target/linux/ath79/patches-5.4/450-fix-block-protection-clearing.patch
index a7ed62302d..dc76161892 100644
--- a/target/linux/ath79/patches-5.4/450-fix-block-protection-clearing.patch
+++ b/target/linux/ath79/patches-5.4/450-fix-block-protection-clearing.patch
@@ -17,12 +17,12 @@ Signed-off-by: Nick Hainke <vincent@systemli.org>

 --- a/drivers/mtd/spi-nor/spi-nor.c
 +++ b/drivers/mtd/spi-nor/spi-nor.c
-@@ -1981,7 +1981,7 @@ static int sr2_bit7_quad_enable(struct s
+@@ -1985,7 +1985,7 @@ static int sr2_bit7_quad_enable(struct s
  static int spi_nor_clear_sr_bp(struct spi_nor *nor)
  {
    int ret;
 -  u8 mask = SR_BP2 | SR_BP1 | SR_BP0;
 +  u8 mask = SR_TB | SR_BP2 | SR_BP1 | SR_BP0;

-   ret = read_sr(nor);
-   if (ret < 0) {
+   if (nor->flags & SNOR_F_HAS_4BIT_BP)
+     mask |= SR_BP3;
```

curtdept commented:
If I were to guess, make prepare is messing with your staging dir and/or tmp dir before it iterates through the targets in the update script. What's your default config target, ath79?

graysky2 commented:
@curtdept - I usually delete `.config` before I run the update script.  In this case, since I needed my tool chain, I copied my ipq806 config over.  The ath config is simply:
```
CONFIG_TARGET_ath79=y
CONFIG_TARGET_ath79_generic=y
CONFIG_TARGET_MULTI_PROFILE=y
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_8dev_carambola2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_8dev_carambola2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_8dev_lima=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_8dev_lima=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_adtran_bsap1800-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_adtran_bsap1800-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_adtran_bsap1840=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_adtran_bsap1840=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_alfa-network_ap121f=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_alfa-network_ap121f=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_alfa-network_ap121fe=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_alfa-network_ap121fe=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_alfa-network_n2q=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_alfa-network_n2q=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_alfa-network_n5q=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_alfa-network_n5q=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_alfa-network_pi-wifi4=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_alfa-network_pi-wifi4=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_alfa-network_r36a=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_alfa-network_r36a=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_allnet_all-wap02860ac=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_allnet_all-wap02860ac=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_arduino_yun=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_arduino_yun=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_aruba_ap-105=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_aruba_ap-105=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_avm_fritz4020=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_avm_fritz4020=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_avm_fritz1750e=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_avm_fritz1750e=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_avm_fritz300e=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_avm_fritz300e=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_avm_fritz450e=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_avm_fritz450e=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_avm_fritzdvbc=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_avm_fritzdvbc=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_buffalo_bhr-4grv=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_buffalo_bhr-4grv=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_buffalo_bhr-4grv2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_buffalo_bhr-4grv2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_buffalo_wzr-600dhp=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_buffalo_wzr-600dhp=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_buffalo_wzr-hp-ag300h=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_buffalo_wzr-hp-ag300h=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_buffalo_wzr-hp-g302h-a1a0=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_buffalo_wzr-hp-g302h-a1a0=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_buffalo_wzr-hp-g450h=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_buffalo_wzr-hp-g450h=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_comfast_cf-e110n-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_comfast_cf-e110n-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_comfast_cf-e120a-v3=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_comfast_cf-e120a-v3=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_comfast_cf-e130n-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_comfast_cf-e130n-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_comfast_cf-e313ac=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_comfast_cf-e313ac=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_comfast_cf-e314n-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_comfast_cf-e314n-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_comfast_cf-e5=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_comfast_cf-e5=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_comfast_cf-e560ac=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_comfast_cf-e560ac=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_comfast_cf-ew72=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_comfast_cf-ew72=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_comfast_cf-wr650ac-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_comfast_cf-wr650ac-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_comfast_cf-wr650ac-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_comfast_cf-wr650ac-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_comfast_cf-wr752ac-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_comfast_cf-wr752ac-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_compex_wpj344-16m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_compex_wpj344-16m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_compex_wpj531-16m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_compex_wpj531-16m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_compex_wpj563=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_compex_wpj563=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_dlink_dap-1330-a1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_dlink_dap-1330-a1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_dlink_dap-1365-a1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_dlink_dap-1365-a1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_dlink_dap-2695-a1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_dlink_dap-2695-a1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_dlink_dch-g020-a1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_dlink_dch-g020-a1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_dlink_dir-505=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_dlink_dir-505=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_dlink_dir-825-b1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_dlink_dir-825-b1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_dlink_dir-825-c1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_dlink_dir-825-c1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_dlink_dir-835-a1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_dlink_dir-835-a1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_dlink_dir-842-c1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_dlink_dir-842-c1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_dlink_dir-842-c2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_dlink_dir-842-c2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_dlink_dir-842-c3=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_dlink_dir-842-c3=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_dlink_dir-859-a1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_dlink_dir-859-a1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_devolo_magic-2-wifi=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_devolo_magic-2-wifi=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_devolo_dvl1200e=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_devolo_dvl1200e=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_devolo_dvl1200i=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_devolo_dvl1200i=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_devolo_dvl1750c=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_devolo_dvl1750c=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_devolo_dvl1750e=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_devolo_dvl1750e=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_devolo_dvl1750i=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_devolo_dvl1750i=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_devolo_dvl1750x=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_devolo_dvl1750x=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_elecom_wrc-1750ghbk2-i=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_elecom_wrc-1750ghbk2-i=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_elecom_wrc-300ghbk2-i=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_elecom_wrc-300ghbk2-i=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_embeddedwireless_dorin=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_embeddedwireless_dorin=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_engenius_ecb1750=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_engenius_ecb1750=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_engenius_enh202-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_engenius_enh202-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_engenius_ens202ext-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_engenius_ens202ext-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_engenius_epg5000=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_engenius_epg5000=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_engenius_ews511ap=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_engenius_ews511ap=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_enterasys_ws-ap3705i=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_enterasys_ws-ap3705i=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_etactica_eg200=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_etactica_eg200=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_glinet_6408=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_glinet_6408=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_glinet_6416=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_glinet_6416=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_glinet_gl-ar150=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_glinet_gl-ar150=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_glinet_gl-ar300m-lite=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_glinet_gl-ar300m-lite=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_glinet_gl-ar300m16=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_glinet_gl-ar300m16=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_glinet_gl-ar750=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_glinet_gl-ar750=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_glinet_gl-mifi=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_glinet_gl-mifi=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_glinet_gl-x750=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_glinet_gl-x750=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_hak5_lan-turtle=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_hak5_lan-turtle=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_hak5_packet-squirrel=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_hak5_packet-squirrel=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_hak5_wifi-pineapple-nano=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_hak5_wifi-pineapple-nano=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_iodata_etg3-r=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_iodata_etg3-r=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_iodata_wn-ac1167dgr=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_iodata_wn-ac1167dgr=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_iodata_wn-ac1600dgr=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_iodata_wn-ac1600dgr=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_iodata_wn-ac1600dgr2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_iodata_wn-ac1600dgr2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_iodata_wn-ag300dgr=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_iodata_wn-ag300dgr=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_jjplus_ja76pf2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_jjplus_ja76pf2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_librerouter_librerouter-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_librerouter_librerouter-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_meraki_mr16=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_meraki_mr16=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_mercury_mw4530r-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_mercury_mw4530r-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_nec_wg1200cr=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_nec_wg1200cr=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_nec_wg800hp=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_nec_wg800hp=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_netgear_ex6400=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_netgear_ex6400=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_netgear_ex7300=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_netgear_ex7300=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_netgear_wndr3700=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_netgear_wndr3700=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_netgear_wndr3700-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_netgear_wndr3700-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_netgear_wndr3800=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_netgear_wndr3800=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_netgear_wndr3800ch=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_netgear_wndr3800ch=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_netgear_wndrmac-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_netgear_wndrmac-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_netgear_wndrmac-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_netgear_wndrmac-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_netgear_wnr2200-16m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_netgear_wnr2200-16m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_netgear_wnr2200-8m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_netgear_wnr2200-8m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ocedo_koala=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ocedo_koala=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ocedo_raccoon=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ocedo_raccoon=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ocedo_ursus=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ocedo_ursus=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_openmesh_om5p-ac-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_openmesh_om5p-ac-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_phicomm_k2t=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_phicomm_k2t=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_pisen_wmm003n=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_pisen_wmm003n=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_pisen_ts-d084=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_pisen_ts-d084=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_pisen_wmb001n=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_pisen_wmb001n=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_pcs_cap324=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_pcs_cap324=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_pcs_cr3000=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_pcs_cr3000=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_pcs_cr5000=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_pcs_cr5000=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_qihoo_c301=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_qihoo_c301=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_qxwlan_e1700ac-v2-16m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_qxwlan_e1700ac-v2-16m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_qxwlan_e1700ac-v2-8m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_qxwlan_e1700ac-v2-8m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_qxwlan_e558-v2-16m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_qxwlan_e558-v2-16m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_qxwlan_e558-v2-8m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_qxwlan_e558-v2-8m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_qxwlan_e600g-v2-16m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_qxwlan_e600g-v2-16m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_qxwlan_e600g-v2-8m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_qxwlan_e600g-v2-8m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_qxwlan_e600gac-v2-16m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_qxwlan_e600gac-v2-16m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_qxwlan_e600gac-v2-8m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_qxwlan_e600gac-v2-8m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_qxwlan_e750a-v4-16m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_qxwlan_e750a-v4-16m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_qxwlan_e750a-v4-8m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_qxwlan_e750a-v4-8m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_qxwlan_e750g-v8-16m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_qxwlan_e750g-v8-16m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_qxwlan_e750g-v8-8m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_qxwlan_e750g-v8-8m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_rosinson_wr818=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_rosinson_wr818=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_samsung_wam250=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_samsung_wam250=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_siemens_ws-ap3610=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_siemens_ws-ap3610=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_sitecom_wlr-7100=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_sitecom_wlr-7100=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_sitecom_wlr-8100=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_sitecom_wlr-8100=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_telco_t1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_telco_t1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_teltonika_rut955=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_teltonika_rut955=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_teltonika_rut955-h7v3c0=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_teltonika_rut955-h7v3c0=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c6-v2-us=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c6-v2-us=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-a7-v5=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-a7-v5=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c2-v3=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c2-v3=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c25-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c25-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c5-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c5-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c58-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c58-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c59-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c59-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c59-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c59-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c6-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c6-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c60-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c60-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c60-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c60-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c60-v3=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c60-v3=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c7-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c7-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c7-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c7-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c7-v4=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c7-v4=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-c7-v5=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-c7-v5=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-d50-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-d50-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-d7-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-d7-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_archer-d7b-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_archer-d7b-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_cpe210-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_cpe210-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_cpe210-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_cpe210-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_cpe210-v3=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_cpe210-v3=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_cpe220-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_cpe220-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_cpe220-v3=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_cpe220-v3=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_cpe510-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_cpe510-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_cpe510-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_cpe510-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_cpe510-v3=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_cpe510-v3=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_cpe610-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_cpe610-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_cpe610-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_cpe610-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_eap225-wall-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_eap225-wall-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_eap245-v3=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_eap245-v3=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_re350k-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_re350k-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_re355-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_re355-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_re450-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_re450-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_re450-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_re450-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_re450-v3=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_re450-v3=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-mr6400-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-mr6400-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wdr3500-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wdr3500-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wdr3600-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wdr3600-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wdr4300-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wdr4300-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wdr4300-v1-il=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wdr4300-v1-il=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wdr4310-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wdr4310-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wdr4900-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wdr4900-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wpa8630-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wpa8630-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wpa8630p-v2-int=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wpa8630p-v2-int=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wpa8630p-v2.0-eu=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wpa8630p-v2.0-eu=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wpa8630p-v2.1-eu=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wpa8630p-v2.1-eu=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr1043n-v5=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr1043n-v5=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr1043nd-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr1043nd-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr1043nd-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr1043nd-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr1043nd-v3=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr1043nd-v3=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr1043nd-v4=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr1043nd-v4=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr1045nd-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr1045nd-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr2543-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr2543-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr710n-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr710n-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr710n-v2.1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr710n-v2.1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr810n-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr810n-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr810n-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr810n-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr842n-v3=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr842n-v3=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr842n-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr842n-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr842n-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr842n-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_tl-wr902ac-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_tl-wr902ac-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_wbs210-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_wbs210-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_wbs210-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_wbs210-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_wbs510-v1=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_wbs510-v1=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_tplink_wbs510-v2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_tplink_wbs510-v2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_trendnet_tew-823dru=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_trendnet_tew-823dru=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_aircube-isp=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_aircube-isp=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_airrouter=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_airrouter=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_bullet-m-ar7240=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_bullet-m-ar7240=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_bullet-m-ar7241=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_bullet-m-ar7241=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_bullet-m-xw=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_bullet-m-xw=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_edgeswitch-5xp=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_edgeswitch-5xp=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_edgeswitch-8xp=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_edgeswitch-8xp=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_lap-120=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_lap-120=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_litebeam-ac-gen2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_litebeam-ac-gen2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_nanobeam-ac=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_nanobeam-ac=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_nanobeam-ac-gen2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_nanobeam-ac-gen2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_nanobridge-m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_nanobridge-m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_nanostation-ac=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_nanostation-ac=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_nanostation-ac-loco=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_nanostation-ac-loco=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_nanostation-loco-m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_nanostation-loco-m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_nanostation-loco-m-xw=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_nanostation-loco-m-xw=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_nanostation-m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_nanostation-m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_nanostation-m-xw=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_nanostation-m-xw=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_picostation-m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_picostation-m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_powerbeam-5ac-500=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_powerbeam-5ac-500=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_powerbeam-5ac-gen2=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_powerbeam-5ac-gen2=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_powerbridge-m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_powerbridge-m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_rocket-m=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_rocket-m=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_routerstation=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_routerstation=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_routerstation-pro=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_routerstation-pro=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_unifi=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_unifi=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_unifiac-lite=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_unifiac-lite=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_unifiac-lr=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_unifiac-lr=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_unifiac-mesh=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_unifiac-mesh=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_unifiac-mesh-pro=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_unifiac-mesh-pro=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_unifiac-pro=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_unifiac-pro=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_ubnt_unifi-ap-pro=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_ubnt_unifi-ap-pro=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_wallys_dr531=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_wallys_dr531=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_wd_mynet-n750=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_wd_mynet-n750=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_wd_mynet-wifi-rangeextender=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_wd_mynet-wifi-rangeextender=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_winchannel_wb2000=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_winchannel_wb2000=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_xiaomi_mi-router-4q=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_xiaomi_mi-router-4q=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_yuncore_a770=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_yuncore_a770=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_yuncore_a782=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_yuncore_a782=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_yuncore_xd4200=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_yuncore_xd4200=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_zbtlink_zbt-wd323=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_zbtlink_zbt-wd323=""
CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_zyxel_nbg6616=y
CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_zyxel_nbg6616=""
CONFIG_ALL_KMODS=y
CONFIG_ALL_NONSHARED=y
CONFIG_TARGET_PER_DEVICE_ROOTFS=y
CONFIG_BUILDBOT=y
CONFIG_COLLECT_KERNEL_DEBUG=y
CONFIG_JSON_OVERVIEW_IMAGE_INFO=y
CONFIG_KERNEL_BUILD_DOMAIN="buildhost"
CONFIG_KERNEL_BUILD_USER="builder"
CONFIG_REPRODUCIBLE_DEBUG_INFO=y
CONFIG_TARGET_ALL_PROFILES=y
# CONFIG_IB is not set
# CONFIG_SDK is not set
```

graysky2 commented:
I guess the question is which is the most correct way to do this?

1. As I have been all along, maintain a git fork, with my own toolchain for building etc. to produce a PR.
2. Since we see this dependency, only on kernel bumps, do it from a fresh clone only like so.  The resulting patch can be applied on top of an existing fork:
```
git clone git@github.com:openwrt/openwrt.git
cd openwrt
mkdir -p staging_dir/host/bin
cp /usr/bin/sed ./staging_dir/host/bin
cp /incoming/openwrt/update_kernel.sh .
./update_kernel.sh -v -u 5.4 5.4.84
git commit -av
git format patch -1
```

Thoughts?

[Here](https://gist.github.com/graysky2/7b0e5986a18246d58b6aabebe184829b) is the diff I got doing it via method 2.

graysky2 commented:
@adschm - The question is which is the most correct way to generate an update to these patches?

1. Running `update_kernel.sh` on an existing fork where users have various toolchains installed, etc.?
2. Since we see this discrepancy, and only on kernel bumps, run the script on a fresh clone only like so.  The resulting patch can be applied to an existing fork:
```
git clone git@github.com:openwrt/openwrt.git
cd openwrt
mkdir -p staging_dir/host/bin
cp /usr/bin/sed ./staging_dir/host/bin
cp /incoming/openwrt/update_kernel.sh .
./update_kernel.sh -v -u 5.4 5.4.84
git commit -av
git format patch -1
```

I am thinking number 2 although more laborious, it rules out any modifications to files by building toolchains or the like.  Thoughts?

[Here](https://gist.github.com/graysky2/7b0e5986a18246d58b6aabebe184829b) is the diff I got doing it via method 2.

graysky2 commented:
@adschm - I updated for 5.4.85 using the 2nd method above.  Can you comment?  As well, since I deleted my fork thus orphaning this PR, I am thinking I will just close this one and send a new one 5.4.83-->5.4.85.  Thoughts on both?

graysky2 commented:
https://github.com/openwrt/openwrt/pull/3706
